cmd/refinery/main.go

```diff
@@ -85,6 +87,11 @@ func main() {
 		os.Exit(0)
 	}
 
+	// instanceID is a unique identifier for this instance of refinery.
+	// We use a hash of a UUID to get a smaller string.
+	h := wyhash.Hash([]byte(uuid.NewString()), 356783547862)
```

Comment:
Is `356783547862` a seed for the hash? Could we add a comment / more somewhere else to make that clearer?